### PR TITLE
Changed animation method on pan gesture now animations are interactiv…

### PR DESCRIPTION
**Pull request by ghost**
Changed UIView.Animate into UIViewPropertyAnimator. Solved jumpy layout when gesture is finished. Sliding animation now is interactive.

### CODE REVIEW

File: Source/SlideOutable.swift [hunk 1]
```diff
@@ -437,16 +437,18 @@ open class SlideOutable: ClearContainerView {
         activeScroll.frame.size.height += antiBounce
         activeScroll.contentInset.bottom += antiBounce
         
-        // Animate to new height
-        UIView.animate(withDuration: 0.5, delay: 0,
-                       usingSpringWithDamping: 0.8,
-                       initialSpringVelocity: velocity.flatMap { abs($0 / (currentOffset - offset)) } ?? 1,
```
Comment (domasn) [inline Source/SlideOutable.swift:443]: Previously we had calculated the initial velocity to match users gesture with animation. Does `UIViewPropertyAnimator` do this out of the box? 😮

File: Example/SlideOutableExample.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -7,6 +7,7 @@
 	objects = {
 
 /* Begin PBXBuildFile section */
+		C0C844A422F400C800DD958A /* SlideOutable.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = D307A2C41DABD2640072E3FB /* SlideOutable.framework */; };
```
Comment (domasn) [inline Example/SlideOutableExample.xcodeproj/project.pbxproj]: All changes in this file do not seem related.

File: Source/SlideOutable.swift (hunk 1 above)
Comment (ghost) [reply Source/SlideOutable.swift:443]: Ping!

**Review by domasn — APPROVED** [review]
👍
